data/beaconrestapi/src/integration-test/java/tech/pegasys/teku/beaconrestapi/AbstractDataBackedRestAPIIntegrationTest.java

```diff
@@ -381,7 +382,12 @@ protected Response post(final String route, final String postData) throws IOExce
 
   protected Response postSsz(final String route, final byte[] postData) throws IOException {
     final RequestBody body = RequestBody.create(postData, SSZ);
-    final Request request = new Request.Builder().url(getUrl() + route).post(body).build();
+    final Request request =
+        new Request.Builder()
+            .url(getUrl() + route)
+            .header(HEADER_CONSENSUS_VERSION, "deneb")
```

Comment:
Added this here temporarily just to see if the plumbing of the header was done right. Won't be part of the final code.